docs/publish/maintain-breaking-changes.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Maintain your Office Add-in
+description: Understand our commitments to compatibility and how to keep your add-in up to date.
+ms.date: 04/29/2022
+ms.localizationpriority: medium
+---
+
+# Maintain your Office Add-in
+
+After you publish your add-in, you should keep it up to date with any important changes from upstream libraries. Patching security issues is critical to building customer trust. Since these changes have no effect on the published manifest, your customers won't need to take any actions to get the latest versions of your add-in.
+
+## Breaking changes in Office.js
+
+The Microsoft 365 Developer Platform is committed to ensuring the compatibility of your add-in. We strive to avoid making breaking changes to the API surface and behavior. However, there are cases were we need to make breaking updates for the sake of security or reliability. In those rare cases, the following steps are taken to ensure users of your add-in are unaffected.
+
+- Announcements that describe the impacted features and recommended changes are made on the [Microsoft 365 Developer Blog](https://devblogs.microsoft.com/microsoft365dev/).
+- If your add-in is published in [AppSource](/office/dev/store/submit-to-appsource-via-partner-center), you'll be contacted through your provided information.
+- Where possible, admins of impacted Microsoft 365 tenants (including [developer tenants](https://developer.microsoft.com/microsoft-365/dev-program)) will be contacted through [Message Center](/microsoft-365/admin/manage/message-center). It's the responsibility of the admin to contact providers of add-in solutions published outside of AppSource.
+
+> [!IMPORTANT]
+> [App Assure](https://www.microsoft.com/fasttrack/microsoft-365/app-assure) guarantees our support to your add-in's compatibility as Microsoft products are updated.
```

Comment:
I'm not sure about saying "to" here. Maybe "of" or "for"...?
```suggestion
> [App Assure](https://www.microsoft.com/fasttrack/microsoft-365/app-assure) guarantees our support of your add-in's compatibility as Microsoft products are updated.
```

Reply:
I reached out to the App Assure folks and they have a much more comprehensive commitment they'd like to have spelled out. Stay tuned :-)